src/_rtmidi.pyx

```diff
@@ -401,6 +448,23 @@ cdef class MidiBase:
             self._port = None
         self.baseptr().closePort()
 
+    def set_error_callback(self, func, data=None):
+        """Register a callback function for errors.
+
+        The callback function is called when an error occurs and must take
+        three arguments. The first argument is a member of enum
+        ``RtMidiError::Type``, represented by one of the ERRORTYPE_* constants.
+        The second argument is an error message. The third argument is the
+        value of the ``data`` argument passed to this function when the
+        callback is registered.
+
+        Registering an error callback function replaces any previously
+        registered error callback.
```

Comment:
By setting a NULL callback, I guess. RtMidi doesn't have an explicit function like `cancelCallback()`, if that's what you mean.